pyscf/lib/dft/libxc_itrf.c

```diff
@@ -383,6 +383,27 @@ double LIBXC_hybrid_coeff(int xc_id)
         return factor;
 }
 
+void LIBXC_nlc_coeff(int xc_id, double *nlc_pars) {
+
+        xc_func_type func;
+        if(xc_func_init(&func, xc_id, XC_UNPOLARIZED) != 0){
+                fprintf(stderr, "XC functional %d not found\n", xc_id);
+                exit(1);
+        }
+        XC(nlc_coef)(&func, &nlc_pars[0], &nlc_pars[1]);
+        xc_func_end(&func);
+}
+
+void LIBXC_rsh_coeff(int xc_id, double *rsh_pars) {
+
+        xc_func_type func;
+        if(xc_func_init(&func, xc_id, XC_UNPOLARIZED) != 0){
+                fprintf(stderr, "XC functional %d not found\n", xc_id);
+                exit(1);
+        }
+        XC(hyb_cam_coef)(&func, &rsh_pars[0], &rsh_pars[1], &rsh_pars[2]);
```

Comment:
This should check if the functional has XC_FLAGS_HYB_CAM.

Reply:
It seemed like LibXC was returning zeros if those parameters weren't defined so I thought this was okay. I might change it.

Reply:
Yes, they should be initialized to zero. But the thing is that there's some functionals that use a different range separation, such as CAMY_BLYP and CAMY_B3LYP that use Yukawa screening.